nativelib/src/main/scala/scala/scalanative/native/package.scala

```diff
@@ -118,4 +119,34 @@ package object native {
     @inline def toUInt: UInt     = toULong.toUInt
     @inline def toULong: ULong   = new ULong(value)
   }
+
+  /** Convert a CString to a java.lang.String */
+  def fromCString(cstr: CString)(implicit charset: java.nio.charset.Charset): String = {
+    val len = string.strlen (cstr).toInt
```

Comment:
Yes, I just mentioned to Eugene like 1hr ago that scalafmt should issue clearer warnings when it fails to format a file. For example, the `sbt scalafmt` task could print

```
[warning] unable to format file path/to/file.scala 
```

I've opened a ticket https://github.com/olafurpg/scalafmt/issues/377